bin/post_compile

```diff
@@ -8,12 +8,29 @@ set -euo pipefail
 # Uses the node binaries/packages installed by the nodejs buildpack previously.
 npm install && npm run build
 
+# We compress files in `build/` below so they can be served more quickly
+# but we want to cache the results for pyodide as it changes relatively
+# frequently and compressing it to brotli is sloooooow
```

Comment:
Do you mean "infrequently"?  (We only update the Pyodide version very few weeks...)

Reply:
Yup, good catch, will fix.